client/src/App.js

```diff
@@ -6,7 +6,6 @@ import Routes from "./components/routing/Routes";
 import setAuthToken from "./utils/setAuthToken";
 import "./App.css";
 import { loadAdmin } from "../src/actions/auth";
-import { UserEditProfile } from "./components/dashboard/UserEditProfile";
 import { loadUser } from "../src/actions/auth";
```

Comment:
Suggestion: loadUser comes from the same file with loadAdmin, you can import it from one place, such as: `import { loadAdmin, loadUser } from "../src/actions/auth";`